docs/conf.py

```diff
@@ -55,6 +55,23 @@
 if 'spelling' in sys.argv:
     extensions.append("sphinxcontrib.spelling")
 
+    from enchant.tokenize import Filter  # noqa
```

Comment:
This is nested because we might not have installed enchant?

Reply:
Yeah, enchant is a dependency of sphinxcontrib-spelling